src/app/organizations/collection/collection.component.html

```diff
@@ -31,7 +31,7 @@
 <app-loading [loading]="(loadingCollection$ | async) || (loadingOrganization$ | async)">
   <div *ngIf="organization$ | async as organization">
     <div class="container mb-5" *ngIf="collection$ | async as collection" fxLayout="column">
-      <div *ngIf="Dockstore.FEATURES.enableNotebooks" class="mt-3">
+      <div class="mt-3">
         <app-preview-warning featureName="Notebooks"></app-preview-warning>
       </div>
```

Comment:
Should we delete this `app-preview-warning` element and the enclosing `div`?  If yes, there's a few other remaining instances of `app-preview-warning` scattered about that might need to be addressed, a search/grep should be able to pinpoint them for you.